specification/_static/custom.css

```diff
@@ -0,0 +1,5 @@
+:root
+img { border-color: black; 
```

Comment:
@PeerHerholz I am not able to make this work on this jupyter book, anything obvious I am missing?

Reply:
Hm, haven't touched the `css` things in a while but IIRC this can be tricky. Did anything change when you built locally or was there no effect at all?

Reply:
nothing changed locally. nope. :-(

Reply:
Ah yes, the classics...Should we maybe `merge` and then I look into things locally on my end?

Reply:
I think we need a different solution here:
![Screenshot from 2022-05-13 07-58-41](https://user-images.githubusercontent.com/83442/168278582-ee1f7ca9-2954-457e-a455-68d99d59199c.png)

Reply:
Pushed a proposal. Apologies for taking the liberty with your branch.

Reply:
hahaha 
no worries